account_move_name_sequence/readme/DESCRIPTION.rst

```diff
@@ -7,6 +7,56 @@ In Odoo version 14.0, the number of journal entries can be manually set by the u
 
+Odoo>=v14.0 raises new concurrency issues since it locks the last journal entry of the journal to get the new number causing a bottleneck
+Even if you only are creating a draft journal entry it locks the last one
+It applies to all accounting Journal Entries
+
+e.g.
+
+ - Customer Invoices,
+ - Credit Notes
+ - Customer Payments
+ - Vendor Bills
+ - Vendor Refunds
+ - Vendors Payment
+ - Manual Journal Entries
+
+Even if your country or company allows relax gaps in a few of these kinds of documents (e.g. only Payments) you are not able to change the implementation to standard allowing gaps of the sequence anymore
+So, all the time it is using no-gap for all journals without an extra table to sequence but locking the last Journal Entry record
+
+Then, the following concurrency errors now are raised frequently:
+
+* Editing the last record used to get the new number from other process
+* Creating a new draft invoice/payment (not only when post it)
+* Creating a transaction to create invoice then payment or vice versa it raises deadlock
+* Reconciling last record since that it could be a heavy process
+* Creating 2 or more Invoices/Bills at the same time
+* Creating 2 or more Payments at the same time
+* Creating 2 or more Journal Entries at the same time
+
+
+All these increases of concurrency errors bring more issues since that Odoo is not prepared:
+
+* Using e-commerce configured with Invoicing Policy Ordered and Automatic Invoice the portal users will see errors in the checkout even if the payment was done the sale order could be in draft state and request a new payment, so double charges
+* Using subscription auto creating invoice after payment you see subscription in payment exception
+* Using accounting creating invoice or payment you will see errors then you will need to start the process again and again until you get the lock before than another user
+* The workers could be used for more time than before since that it could be waiting for release so less concurrent users supported or loading page showed more frequently affecting the performance
+
+The new accounting number is a significant bottleneck
+
+.. image:: https://media.istockphoto.com/vectors/road-highways-with-many-different-vehicles-vector-id1328678690
+
+
+You could not believe all these cases we have created the following unittest to reproduce errors in v14.0 including the deadlock but not v13.0:
+
+ - Passing unittest for `13.0 - [REF] account: Adding unittests for concurrency issues in account_move sequences <https://github.com/odoo/odoo/pull/91614>`_
+ - Concurrency errors for `14.0 - [REF] account: Adding unittests for concurrency issues in account_move sequences <https://github.com/odoo/odoo/pull/91525>`_
+ - `Stress testing and issue reported to Odoo <https://github.com/odoo/odoo/issues/90465>`_
+
+
+Using this module you can decide what kind of documents you can relax to allow gaps or not.
```

Comment:
Using this module, you can configure on what kind of documents the gap sequence may be relaxed